api-errors.json

```
@@ -203,6 +203,10 @@
         "short_message": "Invalid Barcode",
         "long_message": "The barcode could not be created due to invalid barcode type"
     },
+    "40099": {
+        "short_message": "Invitee Inviter",
```

It's a little longer, but much clearer
```suggestion
        "short_message": "Invitee same as Inviter",
```

---

And it would fail the linter which did not agree with our initial "Invitee Is Inviter"  😅